packages/connectivity/connectivity_web/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Is this a different one from the main license?

Reply:
Eeeek, I need to fill it up, I generated this from the plugin template and it doesn't specify any license there.